cmd/server-manager/plugins/events.lua

```diff
@@ -57,16 +90,13 @@ function onRaceWeekendEventSchedule(encodedRaceWeekendSession, encodedRaceWeeken
     -- Encode block, you probably shouldn't touch these either!
     return json.encode(raceWeekend), json.encode(session)
 end
-
--- set this location to the race location, you can download a city list here: http://bulk.openweathermap.org/sample/
-local location = "Manchester,uk"
-
 -- weather API
 function weatherAPI(raceConfig, serverOpts, apiKey)
     -- set the weather based on the current weather at 'location'
     local body, status = httpRequest("http://api.openweathermap.org/data/2.5/weather?q=" .. location .. "&APPID=" .. apiKey, "GET", "")
 
-    if status >= 400 then
+    -- If location not found in openWeatherMap, stop the function and use weather configured in web manager
+    if status == 404 then
```

Comment:
\>=400 is anything bad, they're always failure states (https://www.restapitutorial.com/httpstatuscodes.html). The server will 404 if the location is missing, but it could well error in a bunch of other ways too. Generally safer to just check >=400, as we won't have gotten a workable response with any http response in that range.

Reply:
I had a weird issue, with a location not found by openWeatherMap, the temp was set to 255°C. This is why a set that to == 404

Reply:
That shouldn't make a difference. If a case where == 404 is true then >= 400 is also true.